packages/viewport/src/index.js

```diff
@@ -189,3 +189,32 @@ export function subscribeIsDesktop( listener ) {
 export function getWindowInnerWidth() {
 	return isServer ? SERVER_WIDTH : window.innerWidth;
 }
+
+/******************************************/
+/*       Vertical Scroll Experiment       */
+/******************************************/
+
+//TODO: To be refactored using above using the DESKTOP_BREAKPOINT constant
+export function isTabletResolution() {
+	if ( ! isServer ) {
+		return window.innerWidth < 1040;
+	}
+	return false;
+}
+
+export const DEVICE_MOBILE = 'MOBILE';
+export const DEVICE_TABLET = 'TABLET';
+export const DEVICE_DESKTOP = 'DESKTOP';
+export function resolveDeviceTypeByViewPort() {
+	let deviceType;
+	if ( isMobile() ) {
+		deviceType = DEVICE_MOBILE;
```

Comment:
Really minor, but each of these could be an early return, which would remove the need for the `deviceType` variable as well as each `else` and might make the function slightly more elegant.

```js
if ( isMobile() ) {
  return DEVICE_MOBILE;
}
if ( isTabletResolution() ) {
  return DEVICE_TABLET;
}
return DEVICE_DESKTOP;
```

Reply:
Thanks @sirbrillig that is definitely more elegant 👍 ☺️